src/Microsoft.Health.Dicom.Core/Features/CustomTag/ICustomTagEntryFormalizer.cs

```diff
@@ -0,0 +1,20 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Core.Features.CustomTag
+{
+    /// <summary>
+    /// Formalize custom tag entry.
+    /// </summary>
+    public interface ICustomTagEntryFormalizer
```

Comment:
>ICustomTagEntryFormalizer [](start = 21, length = 25)

Is this something that requires an abstraction like an interface, or is this better off being simply part of the `CustomTagService`?

Reply:
Extract out mostly from unit test perspective. 
It's fine to put into a private method in CustomTagService, but would make it harder to write unit tests.